lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1560,49 +1560,58 @@ def plot(self, xs, ys, *args, **kwargs):
     plot3D = plot
 
     def plot_surface(self, X, Y, Z, *args, **kwargs):
-        '''
+        """
         Create a surface plot.
 
-        By default it will be colored in shades of a solid color,
-        but it also supports color mapping by supplying the *cmap*
-        argument.
-
-        The `rstride` and `cstride` kwargs set the stride used to
-        sample the input data to generate the graph.  If 1k by 1k
-        arrays are passed in, the default values for the strides will
-        result in a 100x100 grid being plotted. Defaults to 10.
-        Raises a ValueError if both stride and count kwargs
```

Comment:
You are also lost the information that the stride and count kwargs can't be used together, and will raise a ValueError.